cdci_data_analysis/analysis/queries.py

```diff
@@ -531,8 +531,9 @@ def get_query_products(self,instrument,job,run_asynch,query_type='Real',logger=N
 
             #status=1
             job.set_failed()
-            if os.environ.get('DISPATCHER_DEBUG', 'yes') == 'yes':
-                raise
+            # TODO are we sure this is still used? Because I instead found DISPATCHER_DEBUG_MODE env
```

Comment:
ok, then can we use DISPATCHER_DEBUG_MODE here?

Reply:
let's see if the tests, also for the plufgins, pass without this, otherwise it's making the debugging and proper execution of the test impossible

Reply:
indeed they're not passing for the `all-sky` plugin